.circleci/config.yml

```diff
@@ -560,7 +560,6 @@ jobs:
     parallelism: 3
     steps:
       - checkout_init_git
-      - halt_if_code_unchanged
```

Comment:
what about the check on puppeteer local? Isn't that more of a concern here?

Reply:
I'm sure what went wrong exactly. `puppeteer-e2e-local-ui` job has been running okay but not for the `puppeteer-e2e-test` job.

Reply:
If we're running on the master branch, we shouldn't be skipping.

When this runs on master commits, it will always generate 0 diffs:
`git diff --name-only $(git merge-base origin/master ${CIRCLE_BRANCH}) `

Previously, the "halt" command would first check whether this branch was master or a release - if so, don't skip. I don't know when that was removed, but it should probably be added back. This change achieves the same thing, though indirectly (since this job *only* runs on master merge).